reflex/components/datadisplay/code.py

```diff
@@ -75,6 +437,11 @@ def create(
         # This component handles style in a special prop.
         custom_style = props.pop("custom_style", {})
 
+        # react-syntax-highlighter doesnt have an explicit "light" theme so we use the default
+        # prism theme to ensure code compatibility
+        if "theme" in props and props["theme"] == "light":
+            props["theme"] = "prism"
```

Comment:
This should be `one-light` instead of `prism` to keep our main website consistent. I think the dark we should map to `one-dark` also